util/ranger/ranger_test.go

```diff
@@ -961,6 +961,7 @@ func TestCompIndexDNFMatch(t *testing.T) {
 	require.NoError(t, err)
 	testKit := testkit.NewTestKit(t, store)
 	testKit.MustExec("use test")
+	testKit.MustExec(`set @@session.tidb_regard_null_as_point=false`)
```

Comment:
What would happen if we don't add this line.

Reply:
Then the default value `true` is used and some plans in this test cases will change.

Reply:
This doesn't affect the correctness, right?